test/test_helper.rb

```diff
@@ -76,6 +82,21 @@ class Application < Rails::Application
     restore_file "config/application.rb"
   end
 
+  # TODO: Refactor existing tests to use this
+  def with_test_suite(test_suite = :minitest, &block)
```

Comment:
I'll write an essay about this some day, but it does not make sense to use default args in private methods (in this case, package-private). You control each call site, so you can pass `:minitest`.

In this case, passing `:minitest` adds clarity at the call site.

Reply:
I want to read this essay!

a46f6dd05ff41f92440d712ac613312c19409ea3